### PR TITLE
adding my profile

### PR DESCRIPTION
**Pull request by RagibAjmal**
Fixes #

## Proposed Changes

  -adding my profile
  -
  -

## Screenshots

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/189"></a>

### CODE REVIEW

File: public/data/ragibajmal3.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Ragib Ajmal",
+    "bio": "I am Ragib Ajmal, associate developer at Accenture ",
```
Comment (schmelto) [inline public/data/ragibajmal3.json:3]:
```suggestion
    "bio": "I am Ragib Ajmal, associate developer at Accenture",
```

File: public/data/ragibajmal3.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    "avatar": "https://github.com/RagibAjmal.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/RagibAjmal",
+        "icon": "github"
+      }
+    ]
+  }
+
```
Comment (schmelto) [inline public/data/ragibajmal3.json:13]:
```suggestion
```

**Review by Nikhil-1503 — CHANGES_REQUESTED** [review]

**Review by FayasNoushad — APPROVED** [review]
Good

**Comment by eddiejaoude** [comment]
@RagibAjmal thank you for the PR, there are some small suggested changes, if you can update, then we can get your PR merged :)

**Review by schmelto — APPROVED** [review]

**Pull request merged**